luacompiler/src/lib/irgen/register_map.rs

```diff
@@ -1,5 +1,8 @@
 use std::{collections::HashMap, vec::Vec};
 
+/// The register in which `_ENV` lives.
+pub const ENV_REG: usize = 0;
```

Comment:
Is it worth using up a register for ENV? We could stick it in a special variable inside the VM?

Reply:
I would say yes, because most of the time you use functions from the standard library. I could introduce new instructions such as `GET/SET_ENV`, or have a special instruction which loads the `_ENV` into a variable, and then use `GET/SET_ATTR` on the register which has `_ENV`.

Reply:
Yes to which question?

Reply:
Oh sorry, yes to the first question. I think I am starting to see your point. But if I special case `_ENV`, that means that I will need special instructions as well, which will have `_ENV` as an implicit argument.

Reply:
Agreed. Let's leave that to a later PR.